docs/security.html

```diff
@@ -677,7 +677,11 @@ <h3 class="anchor-heading"><a id="security_sasl" class="anchor-link"></a><a href
                     <p>To configure SASL authentication on the clients, select a SASL
                         <a href="#security_sasl_mechanism">mechanism</a> that is enabled in
                         the broker for client authentication and follow the steps to configure SASL
-                        for the selected mechanism.</p></li>
+                        for the selected mechanism.</p>
+
+                    <p>Note: Due to how the JRE implements reverse DNS lookups, clients may observe 
+                        slow SASL handshakes if fully qualified domain names are not used when configuring a broker's
```

Comment:
What about adding `When establishing connections to brokers via SASL, clients perform a reverse DNS lookup of the provided broker addresses.` just before?